xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 11.12.1
+JSONRPC_VERSION 11.12.2
```

Comment:
As a backward compatible schema change this need to bump minor not patch e.g. 11.13.0

https://github.com/xbmc/xbmc/blob/04ba4e0fd6d87b5a0fec68b22d96a1049a33128f/xbmc/interfaces/json-rpc/schema/methods.json#L32-L35